Resources/Textures/Objects/Consumable/Drinks/hot_moffee.rsi/meta.json

```diff
@@ -0,0 +1,28 @@
+{
+	"version": 1,
+	"size": {
+		"x": 32,
+		"y": 32
+	},
+	"license": "CC-BY-SA-3.0",
+	"copyright": "https://github.com/discordia-space/CEV-Eris/raw/f7aa28fd4b4d0386c3393d829681ebca526f1d2d/icons/obj/drinks.dmi",
```

Comment:
Copyright needs further attribution. "Taken from (source) at (initial commit)" is common. Be sure the license is something we can use.

Reply:
Noted. It is just a recolour of the coffee sprite, which contained the same attribution. I won't pretend that I know 100% how to attribute it as such, but I'll poke around.

Reply:
```suggestion
	"copyright": "Original Coffee sprite from https://github.com/discordia-space/CEV-Eris/tree/master/icons/obj/drinks.dmi, recolored by OrnamentalFreak (GitHub)",
```